ngraph/frontend/paddlepaddle/src/op/fill_constant_batch_size_like.cpp

```diff
@@ -55,25 +55,51 @@ static std::shared_ptr<Node> set_val(int32_t idx, std::shared_ptr<Node> val_node
     return std::make_shared<ngraph::opset6::Concat>(nodes, 0);
 }
 
+// "str_value" type string, default empty. "value" type float, default 0. They are value to be filled.
+// if "str_value" empty, will fill with "value", else fill with "str_value".
+// reference:
+// https://github.com/PaddlePaddle/Paddle/blob/93d862b0adf224a0af547d1442c57fbd6d0e8efc/paddle/fluid/operators/fill_constant_batch_size_like_op.cc#L41
```

Comment:
I am not sure that it is a good practice to add such comments.

Reply:
okay, I removed this uncertain comments.